crates/gpio/src/backend.rs

```diff
@@ -50,8 +50,18 @@ pub(crate) enum Error {
     DeviceUnimplemented,
 }
 
+macro_rules! gpio_after_help {
+    () => {
+        concat!(
+            "Each device number here will be used to access the corresponding /dev/gpiochipX",
+            "or simulate a GPIO device with N pins (when feature enabled).",
+            " Example, \"-c 4 -l 3:s4:6:s1\"\n",
+        )
+    };
+}
+
```

Comment:
The commit log says that even while testing we need to explicitly enable the feature ? Why is that ? Also your commits have commit version history, you need to drop them ?